### PR TITLE
Put more comments, allow switching backed storage based on env var.

### PR DESCRIPTION
**Pull request by RomanDzhabarov**
@lyft/network-team

1) Cleaner settings + comment
2) Allow select backend storage based on env var (DynamoDB by default)

Was running tests on private repo, all good, i might work at spare time to enable tests on public repo.

### CODE REVIEW

File: app/settings.py [hunk 1]
```diff
@@ -2,38 +2,29 @@
...
     'LOG_LEVEL': 'DEBUG',
-    'SERVER_PORT': 80,
-    'PRETEND_MODE': False,
-    'DYNAMODB_TABLE_HOSTS': DYNAMODB_TABLE_HOSTS,
-    'DYNAMODB_URL': DYNAMODB_URL,
-    'DYNAMODB_CREATE_TABLES_IN_APP': DYNAMODB_CREATE_TABLES_IN_APP,
-    'HOST_TTL': 600,     # 10 minutes
-    'CACHE_TTL': 30
+    'PORT': 80,
+    # Used only in case of DynamoDB backed.
```
Comment (RomanDzhabarov) [inline app/settings.py]: fix typos for backend

File: app/resources/api.py [hunk 2]
```diff
@@ -8,12 +8,32 @@
...
+        storage = settings.value.BACKEND_STORAGE
+
+        if (storage == 'DynamoDB'):
+            return query.DynamoQueryBackend()
+        elif (storage == 'InMemory'):
+            return query.MemoryQueryBackend()
+        elif (storage == 'InFile'):
+            return query.LocalFileQueryBackend()
+        else:
+            raise ValueError('Not supported backend storage: {}'.format(storage))
```
Comment (mattklein123) [inline app/resources/api.py]: Unknown backend storage type specified

File: app/resources/api.py [hunk 3]
```diff
@@ -8,12 +8,32 @@
...
 
 logger = logging.getLogger('resources.api')
 logging.basicConfig(level=logging.DEBUG,
                     format='%(asctime)s %(name)s: %(levelname)s %(message)s')
 
 
+class BackendSelector(object):
+
+    @staticmethod
+    def select():
```
Comment (RomanDzhabarov) [inline app/resources/api.py]: yep, running at startup sounds good

File: app/resources/api.py (hunk 2 above)
Comment (RomanDzhabarov) [inline app/resources/api.py]: fixed

File: app/resources/api.py (hunk 3 above)
Comment (RomanDzhabarov) [inline app/resources/api.py]: fixed

**Comment by mattklein123** [comment]
👍

**Pull request merged**